src/current/cockroachcloud/production-checklist.md

```diff
@@ -70,27 +77,22 @@ Production clusters should not authorize `0.0.0.0/0`, which allows all networks.
+#### CockroachDB {{ site.data.products.serverless }}
+#### CockroachDB {{ site.data.products.dedicated }}
 
-If node termination takes longer than the sum of `server.shutdown.connections.timeout` and `server.shutdown.transactions.timeout`, regardless of the cluster's settings, the node is forcibly terminated and restarted.
+In your application server, set the maximum lifetime of a connection to between 5 and 30 minutes. When a node is shut down or restarted, client connections can be reset after 30 minutes, causing a disruption to applications.
 
-Clusters in CockroachDB {{ site.data.products.cloud }} have the following default values for [cluster settings]({% link {{ site.current_cloud_version }}/cluster-settings.md %}) that relate to [node shutdown]({% link {{ site.current_cloud_version }}/node-shutdown.md %}) for maintenance, upgrades, or scaling. Depending on the requirements of your applications and workloads, you may need to modify `server.shutdown.connections.timeout` or `server.shutdown.transactions.timeout`.
+The following [cluster settings]({% link {{ site.current_cloud_version }}/cluster-settings.md %}) relate to [node shutdown]({% link {{ site.current_cloud_version }}/node-shutdown.md %}) for maintenance, upgrades, or scaling. Depending on the requirements of your applications and workloads, you may need to modify them.
 
-Cluster setting | Default | Details
-----------------|---------|---------
-[`server.shutdown.connections.timeout`]({% link {{ site.current_cloud_version }}/cluster-settings.md %}#setting-server-shutdown-connection-wait)<br />Alias: `server.shutdown.connection_wait` | 0 seconds | How long to wait for client connections to drain before forcibly disconnecting them from the node. Can be set to a value between 0 and 30 minutes (1800 seconds).
-[`server.shutdown.transactions.timeout`]({% link {{ site.current_cloud_version }}/cluster-settings.md %}#setting-server-shutdown-query-wait)<br />Alias: `server.shutdown.query_wait` | 90 seconds | The longest expected duration for an incomplete transaction to complete if it is still running after `server.shutdown.connections.timeout` elapses during node shutdown. After this interval, the transaction will be rolled back to allow the node to restart. A higher value will result in slower cluster operations such as upgrades and scaling events. Consider increasing this value if you experience connection disruptions during these maintenance events. Can be set to a maximum of 90 seconds.
+Cluster setting | Default | Maximum | Details
+----------------|---------|---------|-------------
+[`server.shutdown.connections.timeout`]({% link {{ site.current_cloud_version }}/cluster-settings.md %}#setting-server-shutdown-connection-wait)<br />Alias: `server.shutdown.connection_wait` | 0 seconds  | 3 minutes (1800 seconds) | How long to wait for client connections to drain before forcibly disconnecting them from the node.
```

Comment:
Add: connections with lifetimes over 30m can be interrupted during node restarts.

Reply:
Done with some tweaks. PTAL.